### DISCUSSION TITLE
Update without restarting HA

### DISCUSSION THREAD

**Discussion by oerix** [mampfes/hacs_waste_collection_schedule#2312] [General]
Just wondering is it possible to update WCS without having to restart Home Assistant, every time there is an update?
Maybe a quick restart is possible? I'm guessing no, as it involves sensors? 

Cheers

**Comment by 5ila5** [comment]
Restart after updating is not a WCS specific thing, but it is required to (re-)load every custom_component. HACS will always ask you to restart HA after installing/updating a custom integration. Quick reload only works for only front end integrations so integration that do not add additional functionality but only custom front end widgets/themes

**Comment by oerix** [comment]
Ah okay, thanks for explaining.
